tox.ini

```diff
@@ -40,6 +41,7 @@ conda_deps =
     docs: sphinx_rtd_theme
 conda_channels =
     http://ssb.stsci.edu/astroconda
+    conda-forge
```

Comment:
Not sure if adding conda-forge is a good idea. Adding a channel will likely slow down conda resolver, and may create conflicts with the other channels. (conda-forge recommends using "channel_priority strict" but I'm sure if that's possible with tox-conda https://conda-forge.org/docs/user/tipsandtricks.html#using-multiple-channels). So it may be safer to use pip instead ?